### PR TITLE
Fix fork env for MacOS

### PR DESCRIPTION
**Pull request by mboussaa**

### CODE REVIEW

**Review by fviale — APPROVED** [review]

**Comment by fviale** [comment]
Retest this please

File: app/scripts/proactive/config.js [hunk 1]
```diff
@@ -237,6 +237,8 @@ switch (family) {
     case OperatingSystemFamily.WINDOWS:
         isWindows = true;
         break;
+    case OperatingSystemFamily.MAC:
+        isMac = true;
```
Comment (medou-boushab) [inline app/scripts/proactive/config.js:241]: please add break here

Comment (mboussaa) [reply app/scripts/proactive/config.js:241]: why?

Comment (medou-boushab) [reply app/scripts/proactive/config.js:241]: or do you want isWindows to be false

**Review by medou-boushab — APPROVED** [review]

File: app/scripts/proactive/config.js (hunk 1 above)
Comment (mboussaa) [reply app/scripts/proactive/config.js:241]: I don't want to edit the existing code. Adding a break for mac means I have to declare the isWindows variable outside the switch.

**Pull request merged**